lib/Toc/ToctreeBuilder.php

```diff
@@ -58,6 +68,10 @@ public function buildToctreeFiles(
             }
         }
 
+        if ((bool) ($options['reversed'] ?? false)) {
```

Comment:
Apparently there is a `reversed` option that reverses the whole list. I verified our functionality matches Sphinx, and a test is included.